index.html

```diff
@@ -8,9 +8,42 @@
     <link rel="stylesheet"  href="style.css">
 </head>
 <body>
-    <header>
-        
+    <div class ="porfolio_container">
+    
+    <header >
+        <nav >
+            <ul class ="top_navigation">
+                <li class ="nameplate"> 9:41</li>
+                <ui class ="nameplateicons">
+                <li class ="icons"> <img src = "image/Cellular Connection.png"></li>
+                <li class ="icons"> <img src = "image/Wifi.png"></li>
+                <li class ="icons"> <img src = "image/Battery.png"></li>
+            </ui>
+        </nav>
+        <nav >
+            <ul class ="item_navigation">
+                <li class ="item"><span class= "text"><h2>Omotayo Akande</h2></span></li>
```

Comment:
- [ ] Your logo should be on the left side of the page in the image below, it doesn't look well positioned and it needs to have a link (`a` tag) that redirects a user back to the home page when it's clicked on :+1: 
- Your design
![image](https://user-images.githubusercontent.com/102699958/212214159-cbe2bfbd-81c4-4ab9-9f62-34733b01e9c1.png)
- The Figma design
![image](https://user-images.githubusercontent.com/102699958/212214588-b0b89a91-063b-46ac-bb07-a902b4f79c75.png)